other/guess_the_number_search.py

```diff
@@ -89,11 +89,13 @@ def temp_input_value(
     >>> temp_input_value(min_val=-5100, max_val=-100)
     -5100
     """
-    assert (
-        type(min_val) == int and type(max_val) == int and type(option) == bool
-    ), "Invalid type of value(s) specified to function!"
+    assert isinstance(min_val, int) and isinstance(max_val, int) and \
+        isinstance(option, bool), \
```

Comment:
PEP8: No backslashes in Python code.  Line continuation should be done with ( and ) like it was.

Reply:
If I don't use backslash then the single line length is more than 80 characters, should I use `if` statement with or?

Reply:
assert (
    a,
    b,
    c), "You gotta give me something to work with..."

Reply:
as you suggested, the code is showing the following (flake8)

![Screenshot (234)](https://user-images.githubusercontent.com/78949167/199506849-005c2b15-0200-445a-bb99-90cf0b0c4130.png)

Can I use as showed in following code

```python
assert (
        isinstance(min_val, int) and
        isinstance(max_val, int) and
        isinstance(option, bool)
    ), "Invalid type of value(s) specified to function!"
```

Reply:
Perfect.

Reply:
build/ checks are still failing.